src/user/user.service.ts

```diff
@@ -48,12 +43,36 @@ export class UserService {
     if (!user) {
       throw new HttpException("User not found", HttpStatus.NOT_FOUND);
     }
-
     return { ...user }
-}
+  }
 
-  update(id: number, updateUserDto: UpdateUserDto) {
-    return `This action updates a #${id} user`;
+  async update(id: string, updateUserDto: UpdateUserDto) {
+    if (!id) {
+      throw new HttpException("Missing id", HttpStatus.BAD_REQUEST);
+    }
+    const userExists = await this.prismaService.user.findUnique({
+      where: { id },
+    });
+    if (!userExists) {
+      throw new HttpException(`User not found`, HttpStatus.NOT_FOUND);
```

Comment:
Troca o `User not found` para "User not found" (com aspas duplas) pra ficar padronizado com o resto do código

Reply:
Realizei a alteração.